<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDeviceWindow</class>
 <widget class="QWidget" name="AddDeviceWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>436</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>436</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: #EEE;
border: none;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item alignment="Qt::AlignTop">
    <widget class="QWidget" name="header_frame" native="true">
     <property name="styleSheet">
      <string notr="true">#header_frame {
	background-color: #222831;
	border-top-right-radius: 7px;
	border-top-left-radius: 7px;
}

#header_frame * {
	color: transparent;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="app_icon">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../resources/icons.qrc">:/icons/icons/logo.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="window_title">
        <property name="font">
         <font>
          <family>Montserrat Medium</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #EEE;</string>
        </property>
        <property name="text">
         <string>FitWrap - Add device</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="minimize_window_btn">
        <property name="minimumSize">
         <size>
          <width>15</width>
          <height>15</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">#minimize_window_btn {
	background-color: #00CA4E;
	border-radius: 7px;
	text-align: center;
	margin-bottom: 1px;
}

#minimize_window_btn:hover {
	background-color: #00ff62;
}

#minimize_window_btn:pressed {
	background-color: #1EB859;
}</string>
        </property>
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="close_window_btn">
        <property name="minimumSize">
         <size>
          <width>15</width>
          <height>15</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">#close_window_btn {
	background-color: #FF605C;
	border-radius: 7px;
	text-align: center;
	margin-bottom: 1px;
}

#close_window_btn:hover {
	background-color: #FF8C89;
}

#close_window_btn:pressed {
	background-color: #B7221F;
}</string>
        </property>
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="main_body_frame" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">#main_body_frame {
	background-color: #393E46;
	border-bottom-left-radius: 7px;
	border-bottom-right-radius: 7px;
	border: 2px solid #222831;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item alignment="Qt::AlignHCenter">
             <widget class="QPushButton" name="select_device_icon_btn">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>30</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {	
	color: #222831;
	background-color: #F0CF90;
	border-radius: 5px;
	text-align: center;
}

QPushButton:pressed {
	background-color: #C3943C;
}</string>
              </property>
              <property name="text">
               <string>Select icon (optional)</string>
              </property>
              <property name="icon">
               <iconset resource="../resources/icons.qrc">
                <normaloff>:/icons/icons/folder-svgrepo-com.svg</normaloff>:/icons/icons/folder-svgrepo-com.svg</iconset>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="selected_devic_icon_preview">
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>250</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../resources/icons.qrc">:/icons/icons/default_device_icon.jpg</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <property name="styleSheet">
            <string notr="true">QLineEdit {
	border: 2px solid #40464F;
	border-bottom-color: #F0CF90;
	background-color: #40464F;
	border-radius: 5px;
	padding: 5px;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>30</number>
            </property>
            <property name="bottomMargin">
             <number>30</number>
            </property>
            <item alignment="Qt::AlignBottom">
             <widget class="QLabel" name="brand_label">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string>Brand:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="brand_field"/>
            </item>
            <item alignment="Qt::AlignBottom">
             <widget class="QLabel" name="model_label">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string>Model:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="model_field"/>
            </item>
            <item alignment="Qt::AlignBottom">
             <widget class="QLabel" name="type_label">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string>Type:</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignTop">
             <widget class="QLineEdit" name="type_field"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignBottom">
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item alignment="Qt::AlignVCenter">
          <widget class="QLabel" name="add_device_warning_label">
           <property name="font">
            <font>
             <family>Montserrat Medium</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #FF605C;</string>
           </property>
           <property name="text">
            <string>Fields cannot be empty.</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="add_device_btn">
           <property name="minimumSize">
            <size>
             <width>130</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {	
	color: #222831;
	background-color: #F0CF90;
	border-radius: 5px;
	text-align: center;
}

QPushButton:pressed {
	background-color: #C3943C;
}</string>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
